src/environment/environment.ts

```diff
@@ -1,10 +1,6 @@
 export const environment = {
-    // @ts-ignore
-    production: window['env']['production'],
-    // @ts-ignore
```

Comment:
Can you please revert these changes? I believe there was a rebase/merging issue with this branch.

Reply:
Done